packages/tokens/src/global/color/semantic-code.json

```diff
@@ -0,0 +1,42 @@
+{
+    "color": {
+        "code": {
```

Comment:
[issue/blocker] these tokens don't exist (as variables) in Figma, as far as I know. If we want to use variables for the CSS we should use local Sass variables. Also, these would not be semantic tokens, but component-level tokens.

Plus, it's not clear how these tokens (as in `--token-color`) would work in the context of theming (something [we're still investigating](https://hashicorp.atlassian.net/browse/HDS-4239)). So my suggestion is to keep them local to the file, as `Sass` variables.

(there's an [RFC here](https://docs.google.com/document/d/1Mjoo8FQo3cJWUzf8j3Yh0m4PZoM3Ixw0qwREkgPiKmQ/) but is a bit dated; otherwise happy to jump on a call to explain more in detail how the design tokens are organized)

Reply:
I've just seen @majedelass [comment in Slack](https://hashicorp.slack.com/archives/C025N5V4PFZ/p1733752232153229?thread_ts=1733751911.485639&cid=C025N5V4PFZ) and [this comment in the CRD](https://docs.google.com/document/d/1fmWOUtajtJHsThKG_uozcpSWGJ-O4MmtqzMPEZRqUeg/edit?disco=AAABZPv0994), so I have looked at the file in Figma and I see that the styles and variables seem to be local, not public (@majedelass correct me if I'm wrong):
<img width="243" alt="screenshot_4479" src="https://github.com/user-attachments/assets/49a41ff1-25e3-4c5e-b78e-6e2b14ec042c">
<img width="775" alt="screenshot_4480" src="https://github.com/user-attachments/assets/ee01098a-19a6-43b4-86fa-95976bc7ee3b">

So, this confirms my understanding that these should be local variables (Sass) and not public design tokens.

Happy to discuss more in detail, if needed.

/cc @alex-ju @zamoore @hashicorp/hds-design

Reply:
They are local, you are correct.